### DISCUSSION TITLE
Carrier Sense 100 % ohne Veränderung

### DISCUSSION THREAD

**Discussion by mmaelz** [OpenCCU/OpenCCU#2319] [🇩🇪 Fragen&Antworten] [answered]
Hallo Community, hallo @jens-maus,

nutze RaspberryMatic im HomeAssistant mit dem EQ3-Funkstick. Bis gestern Mittag ging alles ganz wunderbar. Weder am Stick, noch am Standort oder dem Server wurde etwas verändert. Auch der Funkstick liegt seit Monaten an der gleichen Stelle. Nun, seit gestern Nachmittag ist die Carrier Sense dauerhaft bei 100 % und alle 23 Geräte sind nicht mehr erreichbar. 

Konfig: Mini-PC mit Proxmox, Home Assistant 2023.5.3 Supervisor 2023.04.1 Operating System 10.1, RaspberryMatic 3.69.7.20230506

Der Stick wird erkannt: 
Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:07:1b.0-2, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:07:1b.0-2, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK

Hat jemand eine Idee, was ich noch machen kann oder prüfen kann? Stick habe ich sicherheitshalber mal an anderer Stelle platziert. Auch habe ich alle anderen Sticks (SkyConnect, Bluetooth) ausgesteckt. Keine Veränderung.

Was ich nicht verstehe ist der Ausfall von jetzt auf gleich... Danke für Ideen

**Comment by MichaelN0815** [comment, marked as answer]
Siehe https://homematic-forum.de/forum/viewtopic.php?f=31&t=75532

**Reply by mmaelz** [reply]
Vielen Dank. Das habe ich mir natürlich zuerst angeschaut. Aber der CS-Wert ändert sich nicht bei Lageveränderung, es sind weder in meinem Haushalt noch im Haus Funkgeräte hinzu gekommen (Mieter gefragt). Der Wert schwankt auch nicht, sondern ist dauerhaft sofort nach dem Booten auf 100 %

**Reply by MichaelN0815** [reply]
Ja dann. Vielleicht sind es auch einfach Quantenfluktationen.

**Reply by mmaelz** [reply]
Oh. Vielen Dank. Das hilft mir weiter. Werde darauf prüfen. Alles Gute noch

**Comment by jens-maus** [comment]
> nutze RaspberryMatic im HomeAssistant mit dem EQ3-Funkstick. Bis gestern Mittag ging alles ganz wunderbar. Weder am Stick, noch am Standort oder dem Server wurde etwas verändert. Auch der Funkstick liegt seit Monaten an der gleichen Stelle. Nun, seit gestern Nachmittag ist die Carrier Sense dauerhaft bei 100 % und alle 23 Geräte sind nicht mehr erreichbar.

Wenn der CS auf 100% springt hast du definitiv ein massiven Funkstörer des 868MHz Bandes in der näheren Umgebung des HmIP-RFUSB. Neben selbstgemachten Problemen wie einem RaspberryPi der noch danaben liegt, über die Mikrowelle oder andere technische Geräte die ggf. einfach "durchdrehen" und wie wild rumfunken kann es auch ein eigenes HomeMatic oder homematiIP Gerät sein das selbst am Lebensende seiner Batterie steht und daher ggf. einfach ständig neustartet oder andere Probleme macht die dann dazu führen das es die ganze Zeit rumfunkt und damit das funkband dicht macht.

Es gibt aber auch z.B. Erfahrungeberichte wo ein Nachbar irgendein ominöses Gerät das auf dem 868MHz Band funkt betreibt und daher die Funkfrequenz "dicht" macht. Sogar Berichte von achtlos vom Nachbar weggeworfenen "smarten" Heizkörperthermostaten die noch samt Batterie im Müll lagen ist bereits in einigen Foren aufgetreten. Das 868MHz Band ist nunmal ein geshartes Band und genau wie die WLAN Bänder eben auf Kooperation bedacht.

Was ich damit sagen will ist: Da hilft nur eine detaillierte Analyse was da genau den Funkkanal stört. Als erstes hilft es da meistens einfach erst einmal selbst alle seine batteriebetriebenen homematic Geräte komplett stromlos zu machen und zu beobachten (> 1 Stunde) ob sich Besserung einstellt. Wenn nicht dann müsste man mit einem Funk-Analyzer oder ähnlichem auf die Suche gehen um den Übeltäter der Funkstörungen zu finden. Und zu guter letzt kann es natürlich auch der Mini-PC der Proxmox hostet selbst sein und da hilft dann nur via langem USB Verlängerungskabel das noch mit Ferritkernen ausgestattet wurde den Funkstick so weit es geht weg zu bringen oder temporär einen Pi3 einzusetzen und weit weg von anderer Technik zu betreiben um herauszufinden ob es am Proxmox-Host oder einem wilden Funkstörer handelt.

**Comment by friedpa** [comment]
Ich habe mir bei einem ähnlichen Problem einen SDR USB Stick besorgt. Dann alle Geräte stromlos machen, den SDR auf 868MHz Tunen und schauen wo der Übertäter sitzt.

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/23371e41-6931-41c1-8479-ec7b90f40a4d)

**Reply by mmaelz** [reply]
So einen SDR hätte ich tatsächlich auch noch da gehabt. Das wäre bestimmt auch eine super Option zur Fehlersuche gewesen. Danke für den Tipp. Werde ich mir mal merken.

**Comment by mmaelz** [comment]
Vielen Dank, dass hier den Thread nochmals aufgemacht habt.

Ich habe mit einer Anleitung aus dem Homematic Forum meine beiden USB-Sticks an einem frischen Raspi 3B+ getestet und habe die Schwierigkeiten auch dort nachstellen können. Die Carrier Sense war anscheinend hier irreführend. Ich hatte auch noch ein Funkmodul für die GPIOs des Raspi. Mit diesem funktioniert es wieder problemlos.

Den Stick hatte ich ausgeschlossen, weil er ca. 6 Monate problemlos funktionierte, muss nun aber, da der Raspi mit dem Funkmodul in selber Umgebung keine Schwierigkeiten macht, eingestehen, dass beide Sticks hinüber oder zumindest nicht zuverlässig sind.

Danke für eure Aufmerksamkeit

**Reply by jens-maus** [reply]
> Den Stick hatte ich ausgeschlossen, weil er ca. 6 Monate problemlos funktionierte, muss nun aber, da der Raspi mit dem Funkmodul in selber Umgebung keine Schwierigkeiten macht, eingestehen, dass beide Sticks hinüber oder zumindest nicht zuverlässig sind.

Sehr unwahrscheinlich das zwei Sticks komplett defekt sind, korrekt erkannt werden aber beim CarrierSense als folge eines HW defektes hier 100% zeigen solllen. Meine Vermutung wäre eher das dein USB Verlängerungskabel nicht ok ist, du einen rauschenden USB Port hast, kein USB Entstörfilter davorgesetzt hast oder ähnliches und dein Proxmox Host halt einfach selbst dafür sorgt das der HmIP-RF-USB nur rauschen wahrnimmt. Einfach mal den Stick am Pi weit entfernt testen und wenn er da geht ist dein Proxmox Host Schuld…

**Reply by rem-begga** [reply]
Hallo mmaelz,
hast Du den letzten Tipp von Jens noch mal ausprobiert?
Ich habe das geiche Problem mit einem neuen Stick an einem Siemens Esprimo mit Proxmox (Steckt am USB2 Port).

Leider habe ich nur 2 USB-Verlängerungen mit ca. 25 und 50 cm, damit hab ich immer noch den CarrierSense von 100%.

Ein Raspberry Pi4 mit dem GPIO-Board hat nur einen CarrierSense von 2%(Ist bei den Tests mit dem Stick immer ausgeschaltet).
Leider funktionieren meine HM-LANs (sind ebenfalls aktuell stromlos) nicht mehr, so dass ich es damit nicht probieren kann.

Die Lötstellen auf dem Stick hab ich schon nachgelötet und die Firmware ist auch die aktuellste.
Stick wird erkannt, ebenso das Entfernen des Sticks.

Da der Raspberry im Moment nicht bei mir daheim ist kann ich es nicht ausprobieren diesen mit dem Stick zu verbinden.

Viele Grüße
Rudi

**Comment by nilskuehme** [comment]
Ich habe exakt das gleiche Problem. Mein Stick hat sich nicht verändert, was seinen Standort angeht, und dennoch habe ich seit ein paar Wochen sofort 100% nach dem Booten. Ich hatte das im Dezember auch schon Mal und da hat das Einspielen eines Snapshots geholfen. Ich habe leider nicht mehr in Erinnerung welche Snapshotversion das war. Deswegen bin ich mir sicher, dass es diesmal auch kein massiver Störsender sein kann, sondern eben ein softwareseitiges Problem. 

Daher meine Frage, weiß jemand mit welcher Snapshotversion dieses Carrier-Sense-100%-Problem behoben werden kann?

**Reply by MichaelN0815** [reply]
Meines Wissens nach gibt es keinen Beleg dafür, dass es sich hier um ein Software Problem handelt.

**Reply by jens-maus** [reply]
> Deswegen bin ich mir sicher, dass es diesmal auch kein massiver Störsender sein kann, sondern eben ein softwareseitiges Problem.

Das mag man sich natürlich gerne so vorstellen, trotzdem existiert genau dieser Fall nicht. Wenn dort Carrier Sense 100% angezeigt wird ist das 868MHz Funkband nunmal faktisch in der Umgebung der Funkantenne "dicht" und es existiert ein reales Problem das auch in keiner Software zu suchen ist. Da hilft nur auf die Suche des Störsenders gehen und das Problem so in den Griff zu bekommen. RaspberryMatic Versionen upzugraden/downzugraden hilft da definitiv nicht – und wenn, dann nur weil das Hostsystem auf dem RaspberryMatic läuft ggf. der Störsender selber ist und mit einem Neustart oder das Anfassen der Hosthardware dann mitunter das Problem anders/weniger/gar nicht mehr auftreten lässt. Eine softwareseitige Ursache für einen erhöhten CarrierSense gibt es aber definitiv nicht!

**Reply by nilskuehme** [reply]
Da gebe ich Euch recht, da ich selber nicht mehr weiß, bei welcher Snapshotversion das Problem nicht mehr auftrat, habe ich keinen Beleg dafür, dass es sich um ein Softwareproblem handelt. Im Endeffekt ist es ein Erfahrungswert, der aber auch durch sehr viele Faktoren beeinflusst und dadurch falsch sein könnte. Was wäre, wenn ich die alte Version finde und die 100% CS dann doch wieder weg sind? Wäre es dann für euch ein Softwareproblem?

**Reply by MichaelN0815** [reply]
Wenn es reproduzierbar ist - also alte Version CS OK, neue Version CS 100% und man das beliebig switchen kann, dann könnte man es zumindest debuggen. Und das wäre so oder so wertvoll.

**Reply by mmaelz** [reply]
Ich möchte Euch noch meine finale Rückmeldung zu der Problematik geben. Ich hatte ja beschrieben, dass auch ich den Stick weder bewegt noch sonst irgendwie verändert habe und "plötzlich" eine CS von 100 % hatte. Da dies kurz nach einem Update geschah war für mich ebenfalls die Software ursächlich.

Ich habe darauf hin verschiedene Versionen installiert und bin immer bei einer CS von 100 % hängen geblieben ODER die CS war bei 0 % aber Geräte ließen sich nicht anlernen.

Ich holte anschließend meinen alten Raspi3 mit Funkmodul auf dem GPIO aus der Versenkung und tada keine Probleme mit Störungen oder dem Anlernen der Geräte. 

Damit hatte ich meiner Meinung nach zwei Ursachen ausgeschlossen: Störsender im 868 MHz Band, defekte HM oder HMIP Komponenten. Den Funkstick hatte ich, wie verschiedentlich vorgeschlagen, mit unterschiedlichen USB Verlängerungen an unterschiedlichen USB-Ports und zu guter letzt auch am Raspi probiert. Da es mit allen Konstellationen in Verbindung mit dem USB-Stick nicht funktionierte, habe ich den Stick nochmals auseinander genommen und die Lötverbindungen aufgelöst und neu verlötet. 

Was soll ich sagen: angesteckt, eingerichtet am NUC mit Proxmox, erste Komponente angelernt und alles funzt wie am ersten Tag!

Will damit mehrere Dinge sagen: 

1. meine Lötkünste sind offenbar unterirdisch
2.die Software scheidet für mich als Ursache aus
4. man wird alt wie ne Kuh und lernt immer noch dazu

Meine CS dümpelt seit dem neu verlöten auf konstant 0-3 % rum - wobei die meiste Zeit 0 % angezeigt wird.

@rem-begga ich vermute, dass das Erkennen des Sticks absolut nix über die Funktionsfähigkeit des Funkmoduls aussagt. Wenn wir den Stick zusammen löten stellen wir ja IMHO nur die Verbindung zwischen USB-Stick und Funkmodul her. Daher muss eigentlich in dieser Verbindung der Fehler liegen. 

Ich kann nicht erklären, warum die Lötverbindungen nach mind. 6 Monaten unvermittelt nicht mehr gut genug waren. Ich kann auch nicht erklären, warum ich es **nicht** geschafft habe, den neuen Stick gleich richtig zu verlöten (denn ich hatte das Problem mit zwei Sticks). Aber ich hoffe, dass es für andere mit ähnlichen Problemen nochmals eine Hilfestellung ist und ihr den Stick nicht gleich ausschließt, so wie es getan habe. Danke an @jens-maus für die Hilfestellung und die Software und alle Helfer und Mitlesenden.

**Reply by NoLiMiTxo** [reply]
Darauf muss man erstmal kommen. Ich hatte genau das gleiche Problem. Homeassistant mit Raspberrymatic geupdatet und Carrier Sense 100% nach dem update. Lief zuvor mind. 18 Montate am gleichen Standort ohne Probleme.
Standort gewechselt alle anderen Geräte in der umgebung ausgeschaltet. Carrier Sense mal 100% und mal 0% und nicht geht. 

Ich war nun drauf und dran mir einen neuen Stick zu bestellen bis ich gestern dann über diesen Post gestolpert bin. Und siehe da: meine Lötkünste sind genauso unterirdisch. Stick neu gelötet und alles funktioniert wie vor dem Update. Es ist mit nicht erklärlich wieso die Lötverbindungen nach 18+ Monate nichtmehr funktionieren und das genau zeitgleich mit einem Update. Dazu kommt, dass dieser zufall scheinbar nicht nur bei mir eingetreten ist. Das finde ich schon sehr merkwürdigt.

Eigentlich wollte ich nur kurz danke an @mmaelz sagen, dass du deine Lösung noch so ausführlich gepostet hast. Hast mir damit sehr geholfen. Auf die Idee die Verlötung neu zu machen wäre ich vermutlich nie gekommen.

**Comment by rem-begga** [comment]
Hallo zusammen,

nach dem Urlaub hab ich mal die Kommentare durchgeschaut und auch die Infos, welche ich im Forum von Simon42 erhalten habe.
Es sieht wohl so aus als ob meine Lötstellen nicht in Ordnung sind.
Ich habe nun das Angebot von einem guten  Freund angenommen und er testet meinen Stick bei sich und lötet evtl. mal nach, denn sein Stick funktioniert bei ihm und er hat auch sonst die gleiche Kombination wie ich, also Esprimo mit Proxmox.
Es war halt mein erster Lötversuch nach langer Zeit.

Somit können wir alle Fehler am Stick ausschließen.

**Comment by rem-begga** [comment]
Hallo zusammen,

es funktioniert nun bei mir.

Es waren tatsächlich nicht alle Lötpunkte in Ordnung.
Dank einem meiner besten Freunde (studierte Elektrotechnik) sind nun die Lötpunkte in Ordnung und ich habe heute den USB-Stick an den ProxMox angeschlossen.

Carrier Sense ist endlich normal.
Erster Wandthermostat und Radiator-Thermostat sind übernommen worden und vernünftig in Raspberrymatic integriert.
Werde nun die Verbindung zum HomeAssistant herstellen.
Wenn der Durchstich geschafft wurde, werde ich die anderen Heizungsthermostate einbinden.

@mmaelz war mein erste Lötaktion nach ca. 20 Jahren mit einem dementsprechenden Lötkolben.

Danke für all Eure Tipps
und erst Recht an den Löt-.Experten Achim 

Rüdiger

PS: Hab mittlerweile auch endlich eine bessere Lötstation und meine Zisternenanbindung mit D1 Mini klappt (mit meinen Lötkenntnissen)

**Comment by tuxBurner** [comment]
Hi,
  ich hatte auch das Problem.  Carrier Sense war immer auf 100% Anschlag und selten nach nem Reboot auf 0%. Hat aber keine Devices gefunden. Hardware ist ein HMIP-RFUSB Stick von ELV.

Habe noch ne orginale CCU3 rumfliegen da war der Carrier Sense so bei 10%.

Habe dann alle Lötpunkte nachgelötet und nun funktioniert der wieder einwandfrei. 

Finde es spannend das es mehrere Leute gibt bei denen das so spontan auftritt. Der Sticke läuft jetzt seit ca 1.5 Jahre einwandfrei duch.

Eventuell ist das was man mal in der Readme erwähnen sollte ;)
Gruß Seppel

**Comment by Migo84H** [comment]
Hallo zusammen...
So auch gerade bei mir der Fall.
Habe mich seit drei Tagen gewundert, wieso meine Heizkörper nicht mehr durch die Automationen gesteuert werden... kurz in HA/RaspberryMatic reingeschaut, Carrier Sense 100%.
Lief die letzten 2 Jahre problemlos, konnte also fast direkt ausschließen das es an was anderem lag.

Vielen Dank für die vorherigen Posts, dass hat mir meine Fehlersuche extrem erleichtert und erstmal alle Lötpunkte neu gelötet. Zack, dümpelt wieder 0-1% rum.
Herzlichsten DANK!

**Comment by Juergen6762** [comment]
Hallu zusammen,
ich habe mit den Hilfen dieses Artikels ebenfalls an meiner Carrier Sence gebastelt.
Ich habe USB-Verlängerungen probiert für den Stick und nachgelötet und die Lage verändert. Alles ohne Erfolg.
Was letztendlich den Carrier-Sence von 100% auf 2-10% gebracht hat war die Massnahme um das mitgelieferte USB Kabel zum Stick am Ende 3 (!) Ferritkerne auf 30 cm zu verteilen. Hätte ich nicht gedacht. Da gibt es billige passende Kerne, die man als Manschette ca. 3 cm briet um das Kabel Klipsen kann.
Danke für alle Tipps hier. Vielleicht hilft das jemandem weiter.

**Comment by RobeArno** [comment]
Ich möchte noch (vielleicht kuriose) Störer melden:
Waschmaschine und Trockner ;-)

Bei mir ist der Glasfaser-Anschluss im Keller, dort die FritzBox und der Raspberry Pi mit Home Assistant.
Direkt auf einem Regal über Waschmaschine und Trockner.
Es gab immer wieder Störungen, ich wusste nicht warum.

Heute morgen gab es wieder Probleme, als ich einen Raum von der HCU in HomeAssistant überführen wollte. Carrier Sense 100%.
Als ich die Tür vom Trockner geöffnet habe, fiel der Wert auf 70%. Die Waschmaschine lief parallel. Tür geschlossen, Gerät läuft weiter, wieder 100%.

Meinen Raspberry Pi habe ich dann in das Erdgeschoss gebracht und per LAN an einem Repeater angeschlossen. Seitdem ist der CarrierSense wieder auf 1%.

Beide Geräte von Bosch, keine China-Ware.

Mein erster Beitrag, kann nur aufwärts gehen!
